src/composables/trade/useCowswap.ts

```diff
@@ -33,7 +26,7 @@ import { Goals, trackGoal } from '../useFathom';
 
 const HIGH_FEE_THRESHOLD = parseFixed('0.2', 18);
 const APP_DATA =
-  import.meta.env.VITE_GNOSIS_APP_DATA ??
+  process.env.VUE_APP_GNOSIS_APP_DATA ??
```

Comment:
This will break Vite. Why convert it back to proces.env?

Reply:
Sorry, my bad. Fixed.